docs/modules/concepts/pages/user/index.adoc

```diff
@@ -0,0 +1,54 @@
+= James User Model
+:navtitle: User Model
+
+== Operator
+
+== Developer
+
+== James Community
+
+
+
+I am thinking something like:
+
+ * James User --> Operator
+   - Does not compile any code
+   - Does not require an IDE
+   - Has some basic knowledge about email protocols
+   - Has some basic knowledge about how James works (from the docs)
+   - Can perform some relatively simple configurations
+   - Has basic knowledge of mail architecture (protocols), and basic sysadmin 
+skills
+
+This implies:
+   - They do not build a server… must use a provided server
+   - They will only use the provided configurations
+   - They may use the Admin API
+   - They will not use Mailets or any other extensions
+
+By the way, when I originally started the documentation, I also included the 
+roles of “Operator”, “DevOps” etc. With this updated definition of “User” I 
+think that none of these are necessary anymore.
+
+ * User
+   - Just uses email without any notion of what server is providing the service
+
+
+ * Integrator
+   - Develops Mailets / extensions
+   - Uses a lightweight development environment that we provide
+      —> Does not require the core codebase
+
+ * Developer
+   - Anybody who compiles James
+   - Anybody who wants to contribute to James
+   - Anybody who wants to otherwise work with the James codebase
+
+
+A Basic Operator should not be assumed to know much (if anything) about Java and the Java ecosystem.
+
+
+
+Is this another dimension? Basic, Advanced etc.???
```

Comment:
I do not understand the question

Reply:
Well, if "Operator" basically means "doesn't compile" (simplification), and "Developer" basically means "compiles", is there a difference between an Operator who doesn't want to configure anything (target of the Basic Server) and an Operator who is more advanced and doesn't mind getting her hands dirty?

Another thing to consider...

At what point to we force Operators to have knowledge of Java? Just because the system is written in Java, does that mean that the Operator has to know Java?

I am sure other questions will soon arise, too.

Reply:
> At what point to we force Operators to have knowledge of Java?

Enough to run Java.

Basic server operators are expected almost none IMO.

Advanced server operator might need to deal with advanced concepts like Garbage collection and other JVM settings on bigger installations.

Does it makes sense?

Don't hesitate, these questions are valuable.

Reply:
Right now in the docs (not yet committed) I am going with this idea:

* Operator does not require any knowledge of Java
* Integrator must have some expertise in Java, either to do advanced configurations or to develop Extensions.

Roles are not mutually exclusive of course. I will update the text and commit soon so you can tell me what you think.